Packs/CTIX/.secrets-ignore

```diff
@@ -2,4 +2,5 @@ https://abc.com
 https://dummy.com
 https://cyware.com/ctix-stix-taxii-cyber-threat-intelligence-exchange
 https://cyware.com/community/open-apis
-https://cyware.com
\ No newline at end of file
+https://cyware.com
+https://indiaimpex.co/aod/offerTrombley
```

Comment:
Why using this real looking url instead of an example one? When possible we would always prefer example.com.

Reply:
Fixed.